docs/tutorials/tutorial2.md

```diff
@@ -14,148 +14,49 @@ As reported in our paper, we trained an AtacWorks model to learn a mapping from
-
-```
-python $atacworks/scripts/bw2h5.py \
-           --noisybw dsc.1.NK.50.cutsites.smoothed.200.bw \
-           --intervals intervals/hg19.50000.genome_intervals.bed \
-           --out_dir ./ \
-           --prefix NK.50_cells \
-           --pad 5000 \
-           --nolabel
-```
-This creates a file `NK.50_cells.h5`, which contains the noisy ATAC-seq signal to be fed to the pre-trained model.
-
-For more information type `python $atacworks/scripts/bw2h5.py --help`
+See [pre-trained denoising models](pretrained_models.md) for a list of the available pre-trained denoising models.
 
-## Step 7: Inference on selected intervals, producing denoised track and binary peak calls
+Before using one of these models, please read the description of how the training datasets for these models were preprocessed, in [Lal & Chiang, et al. (2019)](https://www.biorxiv.org/content/10.1101/829481v2), Methods section, paragraph 1. If your data processing pipeline is different, it is advisable to train a new model using the instructions above.
 
-```
-python $atacworks/scripts/main.py infer \
-    --files NK.50_cells.h5 \
-    --sizes_file $atacworks/data/reference/hg19.auto.sizes \
-    --config configs/infer_config.yaml \
-    --config_mparams configs/model_structure.yaml \
-```
-Note: `infer_config.yaml` is set up to use multiple GPUs. If you are using a single GPU, edit `infer_config.yaml` to change the line `gpu: "None"` to read `gpu: 0`. 
+See below for instructions to use our pre-trained models or your own trained models.
 
-The inference results will be saved in the folder `output_latest`. This folder will contain four files: 
-1. `NK_inferred.track.bedGraph` 
-2. `NK_inferred.track.bw` 
-3. `NK_inferred.peaks.bedGraph`. 
-4. `NK_inferred.peaks.bw`
+## Input files
 
-`NK_inferred.track.bedGraph` and `NK_inferred.track.bw` contain the denoised ATAC-seq track. `NK_inferred.peaks.bedGraph` and `NK_inferred.peaks.bw` contain the positions in the genome that are designated as peaks (the model predicts that the probability of these positions being part of a peak is at least 0.5)
+To denoise and call peaks from low-coverage/low-quality ATAC-seq data, you need three input files:
 
-To change any of the parameters for inference with the deep learning model, you can edit the parameters in `configs/infer_config.yaml` or `configs/model_structure.yaml` and run the command above. 
+1. A trained AtacWorks model file with extension `.pth.tar`.
 
-Type `python $atacworks/scripts/main.py infer --help` for an explanation of the parameters.
+2. A coverage track representing the number of sequencing reads mapped to each position on the genome in the low-coverage or low-quality dataset. This may be smoothed or processed in the same way as the files used for training the model. Format: [bigWig](https://genome.ucsc.edu/goldenPath/help/bigWig.html)
```

Comment:
Replace "This may be smoothed or processed in the same way as the files used for training the model." with "This track should be processed in the same way as the files used for training the model."